utils/src/state_notifier_gpio_backend.c

```diff
@@ -21,11 +21,12 @@ static inline struct gpio_pin state_to_pin_mapper(enum application_state state)
 {
 	/* *INDENT-OFF* */
```

Comment:
This comment is only relevant for uncrustify, it should be removed or replaced with clang-format comment to disable formatting

Reply:
Done